go/coinstacks/cosmos/api/handlers.go

```diff
@@ -151,43 +186,86 @@ func (h *Handler) GetInfo() (api.Info, error) {
 	return info, nil
 }
 
-func (h *Handler) GetAccount(pubkey string) (api.Account, error) {
-	info, err := h.GetInfo()
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get info")
-	}
+func (h *Handler) GetRawAccountData(pubkey string, denom string, apr *big.Float) (*cosmos.Account, *cosmos.Balance, []cosmos.Delegation, []cosmos.Redelegation, []cosmos.Unbonding, []cosmos.Reward, error) {
 
-	apr, _, err := new(big.Float).Parse(info.(Info).APR, 10)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to parse apr: %s", apr)
-	}
+	var anyError error
 
-	account, err := h.httpClient.GetAccount(pubkey)
-	if err != nil {
-		return nil, err
-	}
+	var wg sync.WaitGroup
+	wg.Add(6)
```

Comment:
so this is gonna exec all 6 requests to the node simultaneously. what happens as we add more? IMO it would be preferable to throttle the number of number of concurrent requesters to a fixed number. i.e. 3 at a time, 4 at a time or whatever and queue beyond that

Reply:
Gonna redo this with ErrorGroup to address this issue and make error handling easier